frontend/web/components/pages/ImportPage.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useEffect, useState } from 'react'
+import _data from 'common/data/base/_data'
+import {
+  useCreateLaunchDarklyProjectImportMutation,
+  useGetLaunchDarklyProjectImportQuery,
+} from 'common/services/useLaunchDarklyProjectImport'
+import AppLoader from 'components/AppLoader'
+
+type ImportPageType = {
+  projectId: string
+  projectName: string
+}
+
+const ImportPage: FC<ImportPageType> = ({ projectId, projectName }) => {
+  const [LDKey, setLDKey] = useState<string>('')
+  const [importId, setImportId] = useState<string>('')
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const [isAppLoading, setAppIsLoading] = useState<boolean>(false)
+  const [projects, setProjects] = useState<string>([])
+  const [createLaunchDarklyProjectImport, { data, isSuccess }] =
+    useCreateLaunchDarklyProjectImportMutation()
+
+  const {
+    data: status,
+    isSuccess: statusLoaded,
+    refetch,
+  } = useGetLaunchDarklyProjectImportQuery({
+    import_id: importId,
+    project_id: projectId,
+  })
+
+  useEffect(() => {
+    const checkImportStatus = async () => {
+      setAppIsLoading(true)
+      const intervalId = setInterval(async () => {
+        await refetch()
+
+        if (statusLoaded && status && status.status.result === 'success') {
+          clearInterval(intervalId)
+          setAppIsLoading(false)
+          window.location.reload()
+        }
+      }, 5000)
```

Comment:
Two questions/concerns here:

1. There's no need to block the UI during import as it's performed asynchronously. It'd be better to display the current import request status.
2. I believe polling interval could be less that 5 seconds as most imports should get completed within it.

Reply:
Precisely because it is taking a few seconds I think that it is a good/acceptable UX to show a blocking loading component during those few seconds, if the task would last around 1 minute or more instead, then I totally agree we should find another non-blocking UX behavior for this.

Reply:
Oh, right. I will lower that 1 sec

Reply:
Done.